src/test/java/io/mosip/mimoto/controller/InjiControllerTest.java

```diff
@@ -290,6 +301,36 @@ public void getIssuerConfigTest() throws Exception {
                 .andExpect(jsonPath("$.errors[0].errorMessage", Matchers.is(API_NOT_ACCESSIBLE_EXCEPTION.getMessage())));
     }
 
+    @Test
+    public void getIssuerWellknownTest() throws Exception {
+        String issuerId = "test-issuer";
+        String wellKnownResponse = "{\"issuer\":\"test-issuer\",\"authorization_endpoint\":\"https://example.com/auth\"}";
+        ResponseEntity<String> wellKnownResponseEntity;
+        wellKnownResponseEntity = restTemplate.getForEntity(wellKnownResponse, String.class);
+        Mockito.when(issuersService.getIssuersWellknown(issuerId)).thenReturn(wellKnownResponseEntity);
+
+        ResponseEntity<Object> resultWellKnownResponseEntity = issuersController.getIssuerWellknown(issuerId);
+        ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
+        responseWrapper.setResponse(wellKnownResponseEntity.getBody());
+        ResponseEntity<Object> wellknownResponseEntityObject = ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
+
+        assertEquals(resultWellKnownResponseEntity, wellknownResponseEntityObject);
+        verify(issuersService, times(1)).getIssuersWellknown(issuerId);
+    }
+
+    @Test
+    public void getIssuerWellknownTestIT() throws Exception {
+        String issuerId = "test-issuer";
+        String wellKnownResponse = "{\"issuer\":\"test-issuer\",\"authorization_endpoint\":\"https://example.com/auth\"}";
+        ResponseEntity<String> wellknownResponseEntity=restTemplate.getForEntity(wellKnownResponse,String.class);
```

Comment:
This is not right.
restTemplate is being  called with issuer wellknown endpoint not with wellknown response